README.md

```diff
@@ -56,10 +56,45 @@
 -->
 [6]: https://codecov.io/gh/seqan/sharg-parser
 
-you can add this library to your application or include the [seqan3 library](https://github.com/seqan/seqan3)
-with even more bio c++ features.
+[7]: https://img.shields.io/badge/license-BSD-green.svg "Open Copyright page"
+<!--
+    This is the Copyright badge link:
+        `"Open Copyright page"` - this text will be shown on hover
+-->
+[8]: https://sharg.vercel.app/usr/html/about_copyright.html
+
+[9]: https://img.shields.io/badge/platform-linux%20%7C%20bsd%20%7C%20osx-informational.svg "Read more about our API"
+<!--
+    This is the API badge link:
+        Redirects to seqan's copyright page.
+        `"Read more about our API"` - this text will be shown on hover
+-->
+[10]: https://sharg.vercel.app/usr/html/about_api.html
+
+[11]: https://img.shields.io/twitter/follow/SeqAnLib.svg?label=follow&style=social "Follow us on Twitter"
+<!--
+    This is the API badge link:
+        Redirects to seqan's copyright page.
+        `"Follow us on Twitter"` - this text will be shown on hover
+-->
+[12]: https://twitter.com/seqanlib
+
+The Sharg argument parser offers a neat and easy-to-use library for argument parsing with C++. With sharg your
+input/output can be validated in a single line, can be customized to your specific needs while providing helpful templates
```

Comment:
The second sentence might need a rephrasing. It is a bit hard to read and I am not sure if it works grammatically.

> With sharg your IO can be (...), can be (...) and makes even subcommanding feel effortless

Reply:
Also in the second text block:
"which not only contains sharg" -> "which does not only contain sharg"
I don't know why, but this sounds better to me